live-editing/configs/StepperConfigGenerator.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    IgxStepperModule,
+    IgxBadgeModule,
+    IgxInputGroupModule,
+    IgxMaskModule,
+    IgxRadioModule,
+    IgxCardModule,
+    IgxButtonModule,
+    IgxButtonGroupModule,
+    IgxCheckboxModule,
+    IgxSelectModule,
+    IgxIconModule
+} from 'igniteui-angular';
+import { AppModuleConfig, Config, IConfigGenerator } from 'igniteui-live-editing';
+
+export class StepperConfigGenerator implements IConfigGenerator {
+
+    public generateConfigs(): Config[] {
+        const configs = new Array<Config>();
+
+        configs.push(new Config({
+            component: 'StepperOverviewSampleComponent',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['StepperOverviewSampleComponent', 'IgxStepperModule', 'IgxMaskModule', 'IgxInputGroupModule', 'IgxButtonModule',
+                    'IgxRadioModule', 'IgxCardModule', 'IgxCheckboxModule', 'IgxSelectModule',
+                    'IgxIconModule', 'IgxBadgeModule'],
+                ngDeclarations: ['StepperOverviewSampleComponent'],
+                ngImports: ['IgxStepperModule', 'IgxMaskModule', 'IgxInputGroupModule', 'IgxButtonModule',
+                    'IgxRadioModule', 'IgxCardModule', 'IgxCheckboxModule', 'IgxSelectModule',
+                    'IgxIconModule', 'IgxBadgeModule']
+            }),
+            shortenComponentPathBy: '/layouts/stepper/'
+        }));
+
+        configs.push(new Config({
+            component: 'StepperLabelPositionAndOrientationSampleComponent',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['StepperLabelPositionAndOrientationSampleComponent',
+                    'IgxStepperModule', 'IgxButtonModule', 'IgxButtonGroupModule'],
+                ngDeclarations: ['StepperLabelPositionAndOrientationSampleComponent'],
+                ngImports: ['IgxStepperModule', 'IgxButtonModule', 'IgxButtonGroupModule']
+            }),
+            shortenComponentPathBy: '/layouts/stepper/'
+        }));
+
+        configs.push(new Config({
+            component: 'StepperLinearSampleComponent',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['StepperLinearSampleComponent', 'IgxStepperModule', 'IgxIconModule', 'IgxButtonModule', 'IgxButtonGroupModule',
+                    'IgxInputGroupModule', 'IgxRadioModule'],
+                ngDeclarations: ['StepperLinearSampleComponent'],
+                ngImports: ['IgxStepperModule', 'IgxIconModule', 'IgxButtonModule',
+                    'IgxButtonGroupModule', 'IgxInputGroupModule', 'IgxRadioModule']
+            }),
+            shortenComponentPathBy: '/layouts/stepper/'
+        }));
+
+        configs.push(new Config({
+            component: 'StepperStepTypesSampleComponent',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['StepperStepTypesSampleComponent', 'IgxStepperModule', 'IgxButtonModule', 'IgxButtonGroupModule'],
```

Comment:
I don't think we need an `IgxButtonModule` here as I don't see any igxButtons being used in the template. The button group imports the button module so it should be safe to remove it in this particular case.